client/v2_2/save_.py

```diff
@@ -168,7 +172,8 @@ def write_file(name, accessor,
       future_to_params[f] = digest_name
 
       layer_name = os.path.join(directory, '%03d.tar.gz' % idx)
-      f = executor.submit(write_file, layer_name, image.blob, blob)
+      message = 'Downloading from {} ({}/{})'.format(image.name(), idx+1, num_layers)
```

Comment:
Downloading from gcr.io/tensorflow/tensorflow:latest (1/12 layers)
Downloading from gcr.io/tensorflow/tensorflow:latest (2/12 layers)
...

maybe more clear?

Reply:
Good idea! I went with slightly modified version:

Downloading from gcr.io/tensorflow/tensorflow:latest (layer 1/12)
Downloading from gcr.io/tensorflow/tensorflow:latest (layer 2/12)
...

Is that ok?

Reply:
Looks great!